hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALSplitter.java

```diff
@@ -220,17 +218,16 @@ public static boolean splitLogFile(Path walDir, FileStatus logfile, FileSystem w
     Path rootDir = CommonFSUtils.getRootDir(conf);
     FileSystem rootFS = rootDir.getFileSystem(conf);
     WALSplitter splitter = new WALSplitter(factory, conf, walRootDir, walFS, rootDir, rootFS);
-    final FileStatus[] wals =
+    final List<FileStatus> wals =
       SplitLogManager.getFileList(conf, Collections.singletonList(walsDir), null);
     List<Path> splits = new ArrayList<>();
-    if (ArrayUtils.isNotEmpty(wals)) {
+    if (CollectionUtils.isNotEmpty(wals)) {
```

Comment:
wals.isEmpty()?

Reply:
The logic here: if wals is not (empty or null), we will do the following loop, so i can't clearly figure out your meaning, could you description it in detail ?